samples/iterative_design_nand.py

```diff
@@ -152,6 +152,7 @@ def run_experiment():
 
     netlist = from_faebryk_t2_netlist(make_t2_netlist_from_t1(t1_))
 
+    Path("./build/faebryk/").mkdir(parents=True, exist_ok=True)
     path = Path("./build/faebryk.net")
```

Comment:
What do you think, should we place everything faebryk generates into the faebryk subfolder?
I think that would make sense. 
Looks like a file which includes global definitions and standards could be nice.
But well for now I'd just change it like this:
```
build_folder_path =  Path("./build/faebryk/")
build_folder_path.mkdir(parents=True, exist_ok=True)
netlist_filepath = build_folder_path / "faebryk.net"
```
And then the same for the savefig stuff
```
plt.savefig(build_folder_path / "graph.png", ...)
```